cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -135,8 +141,8 @@ func (s *server) Shutdown(ctx context.Context) error {
 	if s.server != nil {
 		errs = append(errs, s.server.Close())
 	}
-	if s.jtracer != nil {
-		errs = append(errs, s.jtracer.Close(ctx))
-	}
+	// if s.tracer != nil {
+	// 	errs = append(errs, s.tracer.Shutdown(ctx))
+	// }
```

Comment:
can simplify the whole function to only call server.Close and not use errors.Join